modules/system/utility-wfrp4e.js

```diff
@@ -1132,7 +1132,7 @@ export default class WFRP_Utility {
 
   /** Send effect for owner to apply, unless there isn't one or they aren't active. In that case, do it yourself */
   static async applyOneTimeEffect(effect, actor) {
-    if (game.user.isGM) {
+    if (!game.user.isGM) {
```

Comment:
I don't think this is a typo. It's been a while since I made this but the intention is `if user is GM then check to see if there is a player who owns the actor`

Reply:
Ok, so now i get the intention, and i need to rethink one thing - i was always assuming that socket request will be sent TO gm to elevate permissions for certain actions - and now, its other way round - a GM is sending request to via socket to user and this will break updating the chat message that is created as a communication bridge. I need to check that.

Reply:
hmm, it was so long ago, that it seems i have thought about that anyway :
```
    static updateSocketMessageFlag(data) {
        let message = game.messages.get(data.payload.socketMessageId);
        if (message) {
            if (game.user.isGM) {
                message.delete();
            } else {
                game.socket.emit("system.wfrp4e", { type: "deleteMsg", payload: { "id": message.id } })
            }
        }
    }
```